scalafix-core/shared/src/main/scala/scalafix/config/CustomMessage.scala

```diff
@@ -0,0 +1,47 @@
+package scalafix
+package config
+
+import metaconfig.{Conf, ConfError, ConfDecoder, Configured, Metaconfig}
+import org.langmeta._
+import scalafix.internal.config.MetaconfigPendingUpstream.XtensionConfScalafix
+
+import scala.language.implicitConversions
+
+class CustomMessage[T](
+    val value: T,
+    val message: Option[String],
+    val id: Option[String])
```

Comment:
Can we add `withValue/withMessage/withId` helpers to make it more convenient to copy one field?

Reply:
You only read from the CustomMessage, I don't think this is necessary.

Reply:
Right, I agree.